tests/acceptance/preprocessor-smoke-test-slow.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+var tmp        = require('../helpers/tmp');
+var conf       = require('../helpers/conf');
+var Promise    = require('../../lib/ext/promise');
+var path       = require('path');
+var rimraf     = Promise.denodeify(require('rimraf'));
+var fs         = require('fs');
+var ncp        = Promise.denodeify(require('ncp'));
+var expect     = require('chai').expect;
+
+var runCommand       = require('../helpers/run-command');
+var buildApp         = require('../helpers/build-app');
+var copyFixtureFiles = require('../helpers/copy-fixture-files');
+
+var appName  = 'some-cool-app';
+
+describe('Acceptance: preprocessor-smoke-test', function() {
+  before(function() {
```

Comment:
Probably should use the new pattern for "slow" tests that was introduced in #3013.